textpipe/doc.py

```diff
@@ -274,3 +274,23 @@ def _set_default_nlp(self, lang):
             self._spacy_nlps[lang] = {}
         model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
         self._spacy_nlps[lang][None] = model
+
+    @functools.lru_cache()
+    def sentiment(self, word_map={}):
```

Comment:
I think that it makes sense to have a `sentiment` property and a `compute_sentiment()` method (the latter is what you already have here). The `sentiment` property should call the method with sane defaults. See the `ents` property as an example of this.

This also implies that we actually need (language-specific) sane defaults; now it is possible to call this function without a word map, which will silently return 0 for every text.